docs/configexamples/qos.rst

```diff
@@ -0,0 +1,179 @@
+:lastproofread: 2023-02-18
+
+.. _examples-qos:
+
+#########################
+QOS example
+#########################
+
+Configuration 'dcsp' and shaper using QOS
+=====================================
+
+In this case, we'll try to make a simple lab using QOS and the general ability of the VyOS system.
+Before we recommend you noticed the main article about QOS
+https://docs.vyos.io/en/latest/configuration/trafficpolicy/index.html?highlight=qos#id1
```

Comment:
please change the reference to sphinx style, in this case it should be

:ref:`configuration/trafficpolicy/index:qos`

see [here](https://docs.vyos.io/en/latest/documentation.html#cross-references)